src/vs/code/electron-main/windows.ts

```diff
@@ -727,8 +718,8 @@ export class WindowsManager implements IWindowsMainService {
 			filesToCreate,
 			filesToDiff,
 			filesToWait,
-			forceNewWindow: openInNewWindow,
-			windowToUse
```

Comment:
Passing the target window instance down to the `openInBrowserWindow` method in this file. Note that we fallback to the current window instance only in the case if the `windowToUse` argument is not passed. If the `this.getWindowById(openConfig.targetWindowId)` returns null, we will still open the last active window.